www/caddy/src/opnsense/service/templates/OPNsense/Caddy/rc.conf.d/caddy

```diff
@@ -1,8 +1,39 @@
+{#
```

Comment:
config files usually don't have copyright unless they "program" something complex or difficult. it also helps to see their contents much quicker

Reply:
Removed, thanks.